dojo/api_v2/serializers.py

```diff
@@ -280,7 +281,7 @@ def _pop_tags(self, validated_data):
         return (to_be_tagged, validated_data)
 
 
-class RequestResponseDict(list):
+class RequestResponseDict(UserList):
     def __init__(self, *args, **kwargs):
         pretty_print = kwargs.pop("pretty_print", True)
         list.__init__(self, *args, **kwargs)
```

Comment:
I think this needs the same fix as [here](https://github.com/DefectDojo/django-DefectDojo/pull/11224/files)

```suggestion
        collections.UserList.__init__(self, *args, **kwargs)
```